app/components/govuk_component/header_component.html.erb

```diff
@@ -1,50 +1,45 @@
 <%= tag.header(class: classes, role: 'banner', data: { module: 'govuk-header' }, **html_attributes) do %>
   <div class="govuk-header__container govuk-width-container">
     <div class="govuk-header__logo">
-      <a href=<%= @logo_href %> class="govuk-header__link govuk-header__link--homepage">
+      <%= link_to(@logo_href, class: %w(govuk-header__link govuk-header__link--homepage)) do %>
```

Comment:
To bring parity with the design system, and prefer naming this param `homepage_url` (or if `href` is preferred, `homepage_href`).

Reply:
Or, if using the nested params suggestion below, maybe:

```
logo: {
  crown: true,
  text: 'GOV.UK',
  href: 'https://gov.uk',
}

Reply:
Went for `homepage_url` in the end, it makes sense and matches the Nunjucks.